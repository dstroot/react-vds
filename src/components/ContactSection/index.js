import React from 'react';

// router
import { Link } from 'react-router-dom';
import * as routes from 'constants/routes';

// import SVG files directly as React components. The ReactComponent import
// name is special and tells Create React App that you want a React
// component that renders an SVG, rather than its filename.
import { ReactComponent as Robot } from './media/robot.svg';

const ContactSection = () => (
  <div className="position-relative overflow-hidden p-2 p-md-5 m-md-3 text-center">
    <div className="col-md-7 p-lg-2 mx-auto my-5">
      <h1 className="display-4 text-primary">Get in Touch</h1>
      <p className="lead">
        Verite Data Science is providing customized solutions for both startups
        and Fortune 100 companies. Get in touch to see how we can help.
      </p>
      <div className="svg-container mx-auto">
        <Robot />
      </div>
      <br />
      <Link className="btn btn-lg btn-outline-secondary" to={routes.CONTACT}>
        Contact Us
      </Link>
    </div>
  </div>
);

export default ContactSection;
