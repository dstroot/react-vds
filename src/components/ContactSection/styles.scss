/* SVG Stuff */

.svg-container {
  width: 30%;
  padding-top: 5%;
  margin-bottom: -10%;
  fill: theme-color('primary');
}

.st0 {
  fill: none;
  stroke: theme-color('primary');
  stroke-width: 2;
  stroke-linecap: round;
  stroke-linejoin: round;
  stroke-miterlimit: 10;
}
