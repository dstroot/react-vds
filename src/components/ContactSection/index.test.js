import React from 'react';
import renderer from 'react-test-renderer';

import ContactSection from '../ContactSection';

test('ContactSection renders some content', () => {
  const component = renderer.create(<ContactSection />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
