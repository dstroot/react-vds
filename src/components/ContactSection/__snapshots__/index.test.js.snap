// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactSection renders some content 1`] = `
<div
  className="position-relative overflow-hidden p-2 p-md-5 m-md-3 text-center"
>
  <div
    className="col-md-7 p-lg-2 mx-auto my-5"
  >
    <h1
      className="display-4 text-primary"
    >
      Get in Touch
    </h1>
    <p
      className="lead"
    >
      Verite Data Science is providing customized solutions for both startups and Fortune 100 companies. Get in touch to see how we can help.
    </p>
    <div
      className="svg-container mx-auto"
    >
      <svg>
        robot.svg
      </svg>
    </div>
    <br />
    <a
      className="btn btn-lg btn-outline-secondary"
      href="/contact"
      onClick={[Function]}
    >
      Contact Us
    </a>
  </div>
</div>
`;
